src/pages/workspaces/List.js

```diff
@@ -210,11 +213,11 @@ export const WorkspaceList = ajaxCaller(class WorkspaceList extends Component {
 
   componentDidUpdate() {
     StateHistory.update(_.pick(
-      ['workspaces', 'filter', 'listView'],
+      ['workspaces', 'filter'],
```

Comment:
Does this mean that this setting no longer persists across page refreshes? Are we ok with that?